druid/src/widget/widget_ext.rs

```diff
@@ -303,16 +310,20 @@ mod tests {
         let widget = Slider::new()
             .background(Color::BLACK)
             .align_left()
-            .border(Color::BLACK, 1.0);
+            .border(Color::BLACK, 1.0)
+            .foreground(Color::WHITE);
         assert!(widget.border_is_some());
         assert!(!widget.background_is_some());
+        assert!(!widget.foreground_is_some());
 
         // this should be Container<Slider>
         let widget = Slider::new()
             .background(Color::BLACK)
-            .border(Color::BLACK, 1.0);
+            .border(Color::BLACK, 1.0)
+            .foreground(Color::WHITE);
         assert!(widget.background_is_some());
         assert!(widget.border_is_some());
+        assert!(!widget.foreground_is_some());
```

Comment:
This test is failing because you set it to ! (not), when it is some because you set foreground above. You probably just copied it from above. Removing the ! in the lower one should help.

Reply:
Just changed that one